lib/std/array_list.zig

```diff
@@ -280,14 +298,13 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
         }
 
         /// Increases the array's length to match the full capacity that is already allocated.
-        /// The new elements have `undefined` values. This operation does not invalidate any
-        /// element pointers.
+        /// The new elements have `undefined` values. **Does not** invalidate pointers.
         pub fn expandToCapacity(self: *Self) void {
             self.items.len = self.capacity;
         }
 
         /// Increase length by 1, returning pointer to the new item.
-        /// The returned pointer becomes invalid when the list is resized.
+        /// See struct documentation for returned element pointer validity.
```

Comment:
In general I prefer to put redundant docs in multiple places than to refer the programmer to go read docs somewhere else. For ArrayList, the struct docs you added are nice, but let's also repeat the invalidation warnings on every single function.